libs/tasks/feature/src/lib/tasks-feature/tasks-feature.component.ts

```diff
@@ -1,30 +1,74 @@
-import { Component, OnInit } from '@angular/core';
-import { Task, TasksFacade } from '@todo-workspace/tasks/data-access';
-import { Observable } from 'rxjs';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  OnDestroy,
+  OnInit,
+} from '@angular/core';
+import { TasksFacade } from '@todo-workspace/tasks/data-access';
+import { Observable, Subscription } from 'rxjs';
 import { MatDialog } from '@angular/material/dialog';
 import { TaskFormDialogComponent } from '@todo-workspace/tasks/ui-task-form-dialog';
+import { SnackbarService } from '@todo-workspace/tasks/data-access';
+import { PaginatorState, Task } from '@todo-workspace/tasks/domain';
 
 @Component({
   selector: 'todo-workspace-tasks-feature',
   templateUrl: './tasks-feature.component.html',
   styleUrls: ['./tasks-feature.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class TasksFeatureComponent implements OnInit {
-  tasks$: Observable<Task[]>;
-  constructor(private tasksFacade: TasksFacade, private matDialog: MatDialog) {}
+export class TasksFeatureComponent implements OnInit, OnDestroy {
+  tasks: Task[] = [];
+  readonly todoPage$: Observable<PaginatorState> = this.tasksFacade.todoPage$;
+  readonly donePage$: Observable<PaginatorState> = this.tasksFacade.donePage$;
+  readonly loading$: Observable<boolean> = this.tasksFacade.loading$;
+  errorSubscription: Subscription;
```

Comment:
Visit this page: https://stackoverflow.com/questions/58428791/angular-rxjs-observable-takeuntil-vs-unsubscribe-with-a-subscription
then you can set variable like this:
`readonly errorSubscription$ = this.tasksFacade.error$`